### ISSUE TITLE
44.6 Not Opening

### ISSUE THREAD

**Issue by Captivate9575** [mrvladus/List#18]
Doing `flatpak run io.github.mrvladus.List` prints:
```
Loading tasks...
Add task: misc
Traceback (most recent call last):
  File "/app/share/list/list/application.py", line 57, in do_activate
    Window(application=self).present()
  File "/app/share/list/list/window.py", line 61, in __init__
    self.load_tasks()
  File "/app/share/list/list/window.py", line 88, in load_tasks
    new_task = Task(task, self)
  File "/app/share/list/list/task.py", line 62, in __init__
    self.props.visible = self.task["id"] not in UserData.get()["history"]
KeyError: 'id'
```
And then the app sits in the Gnome background apps dropdown, clicking on it closes it.
Downgrading to the previous version launches fine.
Great app btw :)

**Comment by mrvladus** [comment]
That's strange... I've tested upgrade to new version. Seems like it's not converting to a new format of user data file. Can you try to remove in at ~/.var/app/io.github.mrvladus.List/data/data.json?

**Comment by mrvladus** [comment]
Just make a backup copy before removing

**Comment by Captivate9575** [comment]
Yeah that works, I see the new format for the data.json when adding a test task but its just not applying to the old file, as you said.

**Comment by mrvladus** [comment]
What version are you upgrading from?

**Comment by Captivate9575** [comment]
44.5.4, the problem is the original file didn't specify the version for some reason, it just had `{"version": "", "tasks":` , putting 44.5.4 into that converts it perfectly.

**Comment by mrvladus** [comment]
Found the problem! Thank you!

**Comment by mrvladus** [comment]
Fixed in [44.6.1](https://github.com/mrvladus/List/releases/tag/44.6.1). Update on flathub must be soon.

**Comment by Captivate9575** [comment]
Got the update and it works perfectly, thanks for the quick fix!
